tests/IBTK/nodal_interpolation_01.cpp

```diff
@@ -32,6 +32,8 @@
 // Set up application namespace declarations
 #include <ibtk/app_namespaces.h>
 
+#include "../tests.h"
```

Comment:
Would you mind adding a case where you do not synch up the coarse-fine interface in `HierarchyMathOps::interp()` but instead use `HierarchyGhostCellInterpolation` to do the synch? If `"LINEAR_REFINE"` does the right thing (which it sounds like it doesn't), it ***should*** enforce the hanging node constraints for us so long as the `Variable` is configured so that the coarse values represent the solution. We probably should eventually fix whatever bad thing is happening in the linear refine operator for node data... 😬

Reply:
It does enforce hanging node constraints but the coarse and fine shared nodal values don't match - I added a test. I think this is what we expected to see. At some point I got really bad results with one of these coarsening or refinement operators but I can't remember where I described the issue.